src/ansi-c/ansi_c_parser.cpp

```diff
@@ -215,7 +215,7 @@ Function: ansi_c_parsert::get_class
 
 ansi_c_id_classt ansi_c_parsert::get_class(const typet &type)
 {
-  if(type.id()==ID_typedef)
+  if(type.id()==ID_C_typedef)
```

Comment:
Yes actually, I think so. This function doesn't seem to be called anywhere so the tests didn't cover it, perhaps it should just be deleted?